### PR TITLE
[Azure Stack] Updates to Azs.Update.Admin spec

### PR DESCRIPTION
**Pull request by viananth**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/azsadmin/resource-manager/update/Microsoft.Update.Admin/preview/2016-05-01/UpdateLocations.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "application/json"
   ],
   "paths": {
-    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.Update.Admin/updateLocations/": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Update.Admin/updateLocations/": {
```
Comment (raych1) [inline specification/azsadmin/resource-manager/update/Microsoft.Update.Admin/preview/2016-05-01/UpdateLocations.json:19]: Path change is a breaking change. Can this be released in new API version?

Comment (majastrz) [reply specification/azsadmin/resource-manager/update/Microsoft.Update.Admin/preview/2016-05-01/UpdateLocations.json:19]: ARM paths should be case-insensitive, so this seems fine to me.

---
In reply to: [401298386](https://github.com/Azure/azure-rest-api-specs/pull/8899#discussion_r401298386) [](ancestors = 401298386)

**Review by majastrz — APPROVED** [review]
Signed off from ARM side.

File: specification/azsadmin/resource-manager/update/Microsoft.Update.Admin/preview/2016-05-01/UpdateLocations.json (hunk 1 above)
Comment (bganapa) [reply specification/azsadmin/resource-manager/update/Microsoft.Update.Admin/preview/2016-05-01/UpdateLocations.json:19]: and also the guidance is to follow camelCasing

File: specification/azsadmin/resource-manager/update/readme.md [hunk 2]
```diff
@@ -56,31 +56,3 @@ csharp:
   output-folder: $(csharp-sdks-folder)/Generated
   clear-output-folder: true
 ```
-
```
Comment (bganapa) [inline specification/azsadmin/resource-manager/update/readme.md:59]: why do we remove it from readem, we should keep it here?

**Review by bganapa — CHANGES_REQUESTED** [review]
minor comments

**Review by bganapa — APPROVED** [review]

**Review by raych1 — APPROVED** [review]

**Comment by raych1** [comment]
@akning-ms, please force merge this PR since the case change on path is not a real breaking change. Add @jianyexi , we need to compare path case-insensitively in OAD tool.

**Pull request merged**